packages/bpk-component-definitions-list/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "bpk-component-definitions-list",
```

Comment:
Can we get rid of the plural here (and all other instances...) ? So just `bpk-component-definition-list`